### ISSUE TITLE
PDF creation failed

### ISSUE THREAD

**Issue by henning-gerhardt** [kitodo/kitodo-production#289]
Florian Wille reported following error message on Goobi Community mailing list:

```
ERROR 2014-12-18 16:32:17,952 (GetMetsPdfAction.java:run:328)
         error during pdf generation (java.io.FileNotFoundException)

java.io.FileNotFoundException: 
/usr/local/goobi/metadata/13/images/usr/local/goobi/metadata/13/images/blawvon_672764172_tif/00000004.TIF 
(Datei oder Verzeichnis nicht gefunden)
     at java.io.FileInputStream.open(Native Method)
```

GetMetsPdfAction class is part of delivered WEB-INF/ics-1.1.5.jar file.

**Comment by henning-gerhardt** [comment]
I tried PDF export with a very old Goobi process and its works for me. The main difference between this very old process and new stored processes is that inside mets filesection the path to image files is relative to image directory in old process while new processes contains a absolute path to image files. 

If I manipulate manual path information to relative in new (stored) processes then PDF exports works for this processes too.

Maybe the bug for this is in @matthias-ronge posted source code above: In [line 258](https://github.com/goobi/goobi-production/blob/1.11.x/Goobi/src/de/sub/goobi/export/download/ExportMets.java#L258) are image base path (containing general meta base path including process id and "images" string) and value of stored image path of meta.xml concated to a absolute path.

If image base path is f.e. ``/home/goobi/work/daten/<processid>/images`` and stored value inside meta.xml is ``/home/goobi/work/daten/<processid>/images/<processtitle>/<imagefile>`` then the resulting and later as error message announced string is ``/home/goobi/work/daten/<processid>/images/home/goobi/work/daten/<processid>/images/<processtitle>/<imagefile>`` like @matthias-ronge already showed above. If stored value but only a relative path like ``./<processtitle>/<imagefile>" then is resulting string ```/home/goobi/work/daten/<processid>/images/<processtitle>/<imagefile>`` which works for PDF export.

One question remains: why does this (part of code)[https://github.com/goobi/goobi-production/blob/1.11.x/Goobi/src/de/sub/goobi/export/download/ExportMets.java#L250-260] works correct for any other usage scenario?

**Comment by frank-ulrich-weber** [comment]
In my installation the fileGrp within the meta.xml of a new process contains file-paths like the following:
file:/D:\Goobi_Data\metadata\68825\images\749696575_76787661X_0110_FUW_tif\00000001.tif
This is an absolute path and the PDF is generated without any problem?

**Comment by henning-gerhardt** [comment]
This is strange. Full / absolute path does not work on Unix/Linux based systems but on Windows based? I'm out of ideas...

**Comment by frank-ulrich-weber** [comment]
It is working in my case, because validating the process in class ExportPdf method startExport line 124 fails. In this case the contentServer gets a list of images and not the mets-file with the absolute paths.

**Comment by henning-gerhardt** [comment]
Do you mean: it's works only for invalid documents?

**Comment by frank-ulrich-weber** [comment]
Yes, I think this is a kind of fall-back. You don't get the same pdf, because without the mets-file there is no tabel of contents.

**Comment by matthias-ronge** [comment]
Fixed by #394
